modules/monitoring-creating-a-monitoringstack-object-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-a-monitoringstack-object-for-cluster-observability-operator_{context}"]
+= Creating a MonitoringStack object for the Cluster Observability Operator
+
+To be able to discover the sample `ServiceMonitor` object you created by using the documentation in "Specifying how a service is monitored for Cluster Observability Operator", you must create and configure a `MonitoringStack` object.
```

Comment:
in "Specifying how a service is monitored for Cluster Observability Operator"
should be changed as 
in the "Specifying how a service is monitored for Cluster Observability Operator" section
better provide a link

Reply:
It's not possible to add a link here because of restrictions in the OCP docs repo.